src/pages/AddItem/components/AddItemLists.jsx

```diff
@@ -0,0 +1,86 @@
+import styles from '../styles/AddItemLists.module.css';
+import AddItemImage from './AddItemImage';
+import AddItemTag from './AddItemTag';
+import { useState, useRef } from 'react';
+import { useAddItemForm } from '@/contexts/AddItemFormContext';
+
+export default function AddItemsLists() {
+  const inputRef = useRef(null);
+  const { name, setName, description, setDescription, price, setPrice } = useAddItemForm();
+
+  const handlePriceChange = (e) => {
+    const el = inputRef.current;
+    const rawValue = e.target.value;
+    const cursorPos = el.selectionStart;
+
+    // 쉼표 제거
+    const cleanValue = rawValue.replace(/[^0-9]/g, '');
+    const numericValue = cleanValue === '' ? 0 : Number(cleanValue);
+
+    // 상태 업데이트
+    changePrice(numericValue, rawValue, cursorPos);
+  };
+  const changePrice = (value, rawValue, prevCursorPos) => {
+      setPrice(value === 0 ? '' : value.toLocaleString());
```

Comment:
💊 제안
변수 el, cursorPos의 경우 changePrice 함수에 인수로 사용되기 위한 값이므로 아래처럼 작성하시는 것이
더 가독성 측면에서 좋을 것 같습니다~

```suggestion
  const handlePriceChange = (e) => {
    const rawValue = e.target.value;
    const cleanValue = rawValue.replace(/[^0-9]/g, '');
    const numericValue = cleanValue === '' ? 0 : Number(cleanValue);

    changePrice(numericValue, rawValue);
  };
  
  const changePrice = (value, rawValue) => {
      const prevCursorPos = inputRef.current.selectionStart;
      
      setPrice(value === 0 ? '' : value.toLocaleString());
```